anitya/templates/api.html

```diff
@@ -1,6 +1,6 @@
 {% extends "master.html" %}
 
-{% block title %} API | Anitya {% endblock %}
+{% block title %} API &mdash; Anitya {% endblock %}
```

Comment:
I read an article, **Choosing the best title separator** (https://perishablepress.com/choosing-the-best-title-separators/). There's no specific rule to select a separator over another. It just clears the mess and makes the title readable. I think, we can go with a shorter separator, like: `·` or `-`. What do you say ?